components/stepper/stories/stepper.stories.js

```diff
@@ -1,22 +1,22 @@
 import { Sizes } from "@spectrum-css/preview/decorators";
```

Comment:
```suggestion
import { Sizes, withDownStateDimensionCapture } from "@spectrum-css/preview/decorators";
```

Let's get that in-field button some life when you click it